.gitignore

```diff
@@ -19,3 +19,6 @@ testing/functional/tests/cypress/screenshots/
 testing/functional/tests/cypress/videos/
 testing/functional/tests/cypress/reports/
 testing/functional/tests/cypress/report/
+
+dev/functional/tests/cypress/report
```

Comment:
Apparently now we have a DEV and a `testing` folder on the root of the solution. Any particular reason why we are not using the `testing` folder as discussed since the beginning?

Reply:
Folder location updated as per requirements.